### ISSUE TITLE
Show Notifications with 0 messages displays header visible/cutoff 

### ISSUE THREAD

**Issue by cleidigh** [Microsoft/vscode#45026]
Issue Type: Bug

Show Notifications command (F1/Keybord assignment) when there are no messages
displays a notification header that is cut off.

Presumably it  makes sense to not show anything matching the behavior of clicking the status bar icon.
@bpasero 
Feel free to assign to me...

![image](https://user-images.githubusercontent.com/25272315/36953946-774abbaa-1fec-11e8-969a-0647ebbf6b2b.png)

VS Code version: Code - Insiders 1.21.0-insider (b76a4a71434d6cdee69fa1d8bdc7533d09598d53, 2018-03-02T13:48:32.486Z)
OS version: Windows_NT x64 6.1.7601

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-3615QM CPU @ 2.30GHz (4 x 2294)|
|Memory (System)|7.77GB (0.61GB free)|
|Process Argv|C:\Program Files\Microsoft VS Code Insiders\Code - Insiders.exe|
|Screen Reader|no|
|VM|50%|

</details><details><summary>Extensions (22)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-javascript-snippet-pack|aka|0.1.5
Bookmarks|ale|0.18.1
git-easy|bib|1.9.1
bracket-pair-colorizer|Coe|1.0.15
vscode-eslint|dba|1.4.7
gitlens|eam|8.0.2
tslint|eg2|1.0.28
git-project-manager|fel|1.4.0
file-icons|fil|1.0.7
vscode-npm|fkn|3.3.0
code-runner|for|0.9.0
macros|ged|1.2.1
csharpextensions|jch|1.3.0
csharpfixformat|Leo|0.0.64
start-git-bash|McC|1.2.1
csharp|ms-|1.14.0
debugger-for-chrome|msj|4.1.0
vscode-icons|rob|7.20.0
git-add-remote|sam|0.0.2
code-settings-sync|Sha|2.9.0
gitblame|wad|2.2.0
JavaScriptSnippets|xab|1.5.0

(1 theme extensions excluded)

</details>
Reproduces without extensions

**Comment by bpasero** [comment]
@cleidigh good catch 👍

**Comment by bpasero** [comment]
I pushed a fix to not allow to open the notification center if there are no notifications.

**Comment by isidorn** [comment]
Verified it works nicely. Also did the code review.
@bpasero in theory you could use a when clause for the command.

**Comment by bpasero** [comment]
@isidorn yeah thought about it, but didn't want to push that for stable. There is currently no when clause for when there are 0 notifications in the model.

**Comment by bpasero** [comment]
@cleidigh can you please try this out with latest from master, I pushed a change so that you can open the notifications center also when there are no entries inside:

![image](https://user-images.githubusercontent.com/900690/37393084-61e879b0-2770-11e8-9b9b-9a864389186f.png)

I can no longer see the issue you describe but also do not remember fixing anything there...

**Comment by cleidigh** [comment]
@bpasero 
build new from master this afternoon .  the functionality looks good.
I tried with different screen configurations sides of screen et cetera.  I did not see any of the cut off like I did before.

But one thing I saw is probably by design but yYou are orangeou overflow into the sidebar even if not necessary
see below.  I would like to register a vote to do something to distinguish the icon when no notifications are present
I saw this from another user also FWIW

but to be clear I love the new system

![image](https://user-images.githubusercontent.com/25272315/37436344-e680b172-27bd-11e8-9105-827ce50172e5.png)
